luma/core/cmdline.py

```diff
@@ -131,6 +131,10 @@ def __init__(self, opts, gpio=None):
         self.opts = opts
         self.gpio = gpio
 
+    def noop(self):
```

Comment:
i wouldn't call it noop (there are now 2 `noop`'s in pythons namespace, or overwriting eachother), just call it something like `get_noop`

Reply:
i think it has to be called noop (look at all the other methods in the class: https://github.com/rm-hull/luma.core/blob/feature/framebuffer-device/luma/core/cmdline.py#L125-L214) as it gets instantiated dynamically (see https://github.com/rm-hull/luma.core/blob/feature/framebuffer-device/luma/core/cmdline.py#L241 for example, from the interface given).

Arguably, this should be in a separate PR, it was missing functionality that shouldn't really be in this PR ... i might move it out and do it separately actually

Reply:
sounds good.